eQuiz/modules/eQuiz.Web/eQuiz.Web.csproj

```diff
@@ -89,19 +89,27 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Areas\Admin\Scripts\equiz.layoutController.js" />
+    <Content Include="Areas\Admin\Scripts\review\equiz.reviewController.js" />
+    <Content Include="Areas\Admin\Scripts\review\equiz.reviewDataService.js" />
+    <Content Include="Areas\Admin\Scripts\shared\equiz.module.js" />
+    <Content Include="Areas\Admin\Styles\bootstrap-combined.css" />
     <Content Include="Areas\Admin\Styles\equiz-temp.css" />
+    <Content Include="Areas\Admin\Views\Pages\Review.html" />
     <Content Include="Areas\Moderator\Styles\equiz-temp.css" />
     <Content Include="Areas\Student\Styles\equiz-temp.css" />
     <Content Include="Global.asax" />
     <Content Include="Scripts\app\shared\equiz.module.js" />
     <Content Include="Scripts\libs\angularjs\angular.js" />
     <Content Include="Scripts\libs\angularjs\angular.min.js" />
+    <Content Include="Scripts\libs\Underscore\underscore-min.js" />
```

Comment:
The reason behind refusing underscorejs is that it does not fit AngularJS architecture.
In case we cannot avoid using it we need to employ angularjs-underscorejs adapter.